torch/backends/cudnn/__init__.py

```diff
@@ -102,15 +112,16 @@ def is_acceptable(tensor):
         return False
     if not is_available():
         warnings.warn(
-            "PyTorch was compiled without cuDNN support. To use cuDNN, rebuild "
+            "PyTorch was compiled without cuDNN/MIOpen support. To use cuDNN/MIOpen, rebuild "
             "PyTorch making sure the library is visible to the build system.")
         return False
     if _libcudnn() is None:
-        warnings.warn('cuDNN library not found. Check your {libpath}'.format(
-            libpath={
-                'darwin': 'DYLD_LIBRARY_PATH',
-                'win32': 'PATH'
-            }.get(sys.platform, 'LD_LIBRARY_PATH')))
+        if cuda is not None:
```

Comment:
I believe this warning applies to MIOpen as well?

Reply:
I think you're right.